### PR TITLE
Cleanup pipeline-lite deployment

### PR DESCRIPTION
**Pull request by IronPan**
- Update gcp deployment to have a single place for configuration. Add new readme
- Update the high level readme
- Remove unnecessary namespace overlay files.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1921)

### CODE REVIEW

File: manifests/kustomize/README.md [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
-(Or role creator in your namespace)
+Then open the Pipeline main page
 ```
-kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=[your-user-name]
-```
-
-When deployment is complete, you can access Kubeflow Pipelines UI by an IAM controlled public endpoint, which can be found by
-```
-kubectl describe configmap inverse-proxy-config -n kubeflow
+open http://$(kubectl describe configmap inverse-proxy-config -n kubeflow | grep googleusercontent.com)
```
Comment (ajchili) [inline manifests/kustomize/README.md]: `open` is only available on macOS

File: manifests/kustomize/README.md [hunk 2]
```diff
@@ -51,7 +50,7 @@ and open http://localhost:8080/
 
 
 
-# Uninstall
+## Uninstall
 You can uninstall Kubeflow Pipelines by running
 ```
 kubectl delete -f https://raw.githubusercontent.com/kubeflow/pipelines/$PIPELINE_VERSION/manifests/kustomize/namespaced-install.yaml
```
Comment (ajchili) [inline manifests/kustomize/README.md]: `export PIPELINE_VERSION=0.1.26` should also be add within this code block. A user may not uninstall their deployment right away.

File: manifests/kustomize/README.md (hunk 1 above)
Comment (IronPan) [reply manifests/kustomize/README.md]: done

File: manifests/kustomize/README.md (hunk 2 above)
Comment (IronPan) [reply manifests/kustomize/README.md]: done

**Pull request merged**